acceptance/windows-service/test/integration/chef-windows-service/inspec/chef_windows_service_spec.rb

```diff
@@ -0,0 +1,58 @@
+only_if do
```

Comment:
I haven't seen `only_if` before - is that an inspec feature or an rspec feature?